core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -382,6 +402,29 @@ impl SchedulerController {
             );
         }
     }
+
+    /// Calculate prioritization for a transaction.
+    fn calculate_prioritization(
+        fee_structure: &FeeStructure,
+        include_loaded_account_data_size_in_fee: bool,
+        message: &SanitizedMessage,
+        budget_limits: &FeeBudgetLimits,
+        cost: u64,
+    ) -> u64 {
+        let total_fee = fee_structure.calculate_fee(
+            message,
+            5_000, // this just need to be non-zero
```

Comment:
I read the called code... i won't insist for yet another prep pr for cleaning up this api mess. ;)